src/analyses/commonPriv.ml

```diff
@@ -126,16 +126,16 @@ struct
   module RequireMutexActivatedInit =
   struct
     let init () =
-      let analyses = GobConfig.get_list "ana.activated" in
-      let mutex_active = List.exists (fun x -> Json.string x="mutex") analyses || List.for_all (fun x -> Json.string x<>"base") analyses in
+      let analyses = GobConfig.get_string_list "ana.activated" in
+      let mutex_active = List.mem "mutex" analyses || not (List.mem "base" analyses) in
```

Comment:
Nice catch, my code was a bit over complicated here :roll_eyes:

Reply:
Surprisingly it was complicated like this everywhere. I guess when `get_string_list` was added for convenience, the existing uses of `get_list` weren't reviewed.